apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/config.py

```diff
@@ -89,6 +89,10 @@
     cfg.BoolOpt('single_tenant_mode', default=False,
                 help=_("All the Openstack tenants will be described by a "
                        "single ACI tenant.")),
+    cfg.StrOpt('shared_tenant_name',
+               default=None,
```

Comment:
Can the default be set to ${apic_system_id}. Then you can avoid having conditional code everywhere. Also to be consistent with 'single_tenant_mode', should this option be named 'single_tenant_name'? Will avoid confusion and mistakes.